src/Chart.php

```diff
@@ -184,19 +215,41 @@ public function setModel(Model $model, array $columns = [], array $stacks = []):
      *
      * @return $this
      */
-    public function withCurrency(string $char = '€', string $axis = 'y')
+    public function setCurrencyLabel(string $char = '€', string $axis = 'y')
     {
         // magic regex adds commas as thousand separators: http://009co.com/?p=598
-        $options = [];
-        $options['scales'][$axis . 'Axes'] =
-            [['ticks' => [
-                'userCallback' => new JsExpression('{}', ['function(value) { value=Math.round(value*1000000)/1000000; return "' . $char . ' " + value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ","); }']),
-            ]]];
-
-        $options['tooltips'] = [
-            'enabled' => true,
-            'mode' => 'single',
-            'callbacks' => ['label' => new JsExpression('{}', ['function(item, data) { return item.' . $axis . 'Label ? "' . $char . ' " +  item.' . $axis . 'Label.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",") : "No Data"; }'])],
+        $options = [
+            'scales' => [
+                $axis => [
+                    'ticks' => [
+                        'callback' => new JsExpression('{}', [
+                            'function(value, index, ticks) {
+                                value = Math.round(value*1000000)/1000000;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision

Reply:
Can't use toPrecision because i need 6 significant digits after dot, but toPrecision works differently:
```
Number(123.456).toPrecision(4) // gives 123.4 not 123.4560 as I would expect
```

Reply:
why rounding the chart data at all?

Reply:
Actually decided to use toLocaleString instead. It does all we need - rounding and localization.